File: Android/app/src/main/java/com/denarii/android/user/CreditCard.java
package com.denarii.android.user;

import java.io.Serializable;

public class CreditCard implements Serializable {
  private String customerID;

  private String sourceTokenID;

  private boolean hasCreditCardInfo = false;

  public void setCustomerID(String newID) {
    customerID = newID;
  }

  public void setSourceTokenID(String newSourceID) {
    sourceTokenID = newSourceID;
  }

  public void setHasCreditCardInfo(boolean hasCreditCardInfo) {
    this.hasCreditCardInfo = hasCreditCardInfo;
  }

  public String getCustomerID() {
    return customerID;
  }

  public String getSourceTokenID() {
    return sourceTokenID;
  }

  public boolean getHasCreditCardInfo() {
    return hasCreditCardInfo;
  }
}

File: Ios/denarii/denarii/user/CreditCard.swift
//
//  CreditCard.swift
//  denarii
//
//  Created by Andrew Katson on 9/30/23.
//

import Foundation

class CreditCard {

    var customerId : String = ""
    var sourceTokenId : String = ""
    
    init(customerId: String, sourceTokenId: String) {
        self.customerId = customerId
        self.sourceTokenId = sourceTokenId
    }
    
    init() {
        self.customerId = ""
        self.sourceTokenId = ""
    }
}
